hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/replication/UnhealthyReplicationProcessor.java

```diff
@@ -117,19 +114,27 @@ public void processAll(ReplicationQueue queue) {
         healthStateCntMap.compute(healthResult.getHealthState(),
                 (healthState, cnt) -> cnt == null ? 1 : (cnt + 1));
       } catch (Exception e) {
-        LOG.error("Error processing Health result of class: {} for " +
-                   "container {}", healthResult.getClass(),
-                healthResult.getContainerInfo(), e);
-        failed++;
+        if (e instanceof CommandTargetOverloadedException) {
```

Comment:
Nit: I'd prefer separate `catch` blocks, even if `failedOnes.add` needs to be duplicated.